Bindings/Java/OpenSimJNI/About.cpp

```diff
@@ -36,7 +36,7 @@
 #define STR(var) #var
 #define MAKE_VERSION_STRING(maj,min,build)  STR(maj.min.build)
 #define MAKE_COPYRIGHT_STRING(y,a) \
-    "Copyright (c) " STR(y) " Stanford University, " STR(a)
+    "Copyright (c) 2005-2016 Stanford University, " STR(a)
```

Comment:
I would prefer to roll back this change here. Otherwise, I think the macro should be updated to not have an unused parameter.